pages/common/nvim.md

```diff
@@ -1,7 +1,7 @@
 # nvim
 
 > Neovim, a programmer's text editor based on Vim, provides several modes for different kinds of text manipulation.
-> Pressing `i` enters edit mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion.
+> Pressing while in normal mode `i` enters insert mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion.
```

Comment:
```suggestion
> Pressing `i` in normal mode enters insert mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion.
```
I think this would be better.

Reply:
Sure thing don't have any objections against the suggestion.